include/azure_kinect_ros_driver/k4a_exposure_calibration.h

```diff
@@ -0,0 +1,158 @@
+// PlusOne Robotics
+// Author: Shannon Stoehr
+// email:  shannon.stoehr@plusonerobotics.com
+
+#pragma once
+
+// Library headers
+#include <ros/ros.h>
+#include <sensor_msgs/Image.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <opencv2/opencv.hpp>
+#include <dynamic_reconfigure/client.h>
+#include <k4a/k4a.h>
+#include <mutex>
+
+// Project headers
+#include "azure_kinect_ros_driver/AzureKinectParamsConfig.h"
+#include "azure_kinect_ros_driver/k4a_update_exposure.h"
+#include "azure_kinect_ros_driver/k4a_auto_tune_exposure.h"
+#include "azure_kinect_ros_driver/k4aCameraExposureServiceErrorCode.h"
+
+/**
+ * @brief Class that allows the user to tune exposure in real time from the command line
+ */
+class K4AExposureCalibration
+{
+public:
+    /** @brief default constructor and destructor*/
+    K4AExposureCalibration();
```

Comment:
I'd remove the default constructor so as not to allow it. It doesn't make much sense without a NodeHandle anyway.

Reply:
So, unfortunately for now these are required in order for the current catkin testing setup to work.